packages/atomic/src/locales.json

```
@@ -5265,6 +5265,33 @@
     "zh-CN": "与本案有关的见解",
     "zh-TW": "與此案件相關的洞見"
   },
+  "viewed-by-customer": {
+    "en": "Viewed by customer",
+    "fr": "Vu par le client",
+    "cs": "Zobrazeno zákazníkem",
```

I think this should be zákazníkovi (by the customer)
```suggestion
    "cs": "Zobrazeno zákazníkovi",
```